api-server/server/routes/remoteApis.js

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env node
+'use strict'
+
+const LOGGER = require("../logger").logger
+const express = require("express")
+const openapiSampler = require('openapi-sampler');
+const refParser = require('json-schema-ref-parser');
+const { api, event, connection } = require("@varkes/app-connector")
+module.exports = {
+    router: router
+}
+
+function getAll(req, res) {
+    LOGGER.debug("Getting all APIs")
+    var err = assureConnected()
+    if (err) {
+        res.status(400).send({ error: err })
+    } else {
+        api.findAll().then((result) => {
+            res.status(result.statusCode).send(result.body);
+        })
+    }
+}
+
+function get(req, res) {
+    LOGGER.debug("Get API %s", req.params.api)
+    var err = assureConnected()
+    if (err) {
+        res.status(400).send({ error: err })
+    } else {
+        api.findOne(req.params.api).then((result) => {
+            if (result.statusCode < 400) {

```

Comment:
Not related to the task but still can you please refactor that method a bit, I always loose focus when reading it. Just introduce some helper functions with good name

Reply:
done